### PR TITLE
improvements for new cli

### PR DESCRIPTION
**Pull request by dboskovic**

### CODE REVIEW

**Comment by carolshark** [comment]
@dboskovic, considering this branch is being used actively in production, I suggest merging it to master already. Also from here on we can start labeling versions used by the plugins. This can be 1.0.0

File: lib/response.js [hunk 1]
```diff
@@ -30,35 +30,36 @@ Response = (function () {
...
+      this._respond(data || {});
     }
   };
   Response.prototype.job_fail = function (status, msg, data) {
     this.meta('set_job_status', 'failed');
     this.meta('set_job_status_message', status);
     this.meta('set_job_failure_message', msg);
     if(data) {
-      this._respond(data || true);
+      this._respond(data || {});
```
Comment (carolshark) [inline lib/response.js]: shouldn't this call pass status and method name as 'fail?

Comment (dboskovic) [inline lib/response.js]: vs `failed` ?

Comment (carolshark) [inline lib/response.js]: @dboskovic I mean the method name parameter. It's used only in error method, but not job failed

**Pull request merged**